### PR TITLE
Remove needle check in console glibc_i686 test

### PR DESCRIPTION
**Pull request by dzedro**
Get rid of assert in console test

[test](http://10.100.12.155/tests/5529)

### CODE REVIEW

File: tests/console/glibc_i686.pm [hunk 1]
```diff
@@ -1,30 +1,28 @@
...
 
-    assert_script_run("zypper -n in -C libc.so.6", 500);
-
-    # select user console for our needles to match
-    select_console 'user-console';
-    script_run("/lib/libc.so.*", 0);
-    assert_screen 'test-glibc_i686-1';
+    zypper_call 'in -C libc.so.6';
+    script_run("/lib/libc.so.* | tee /dev/$serialdev", 0);
+    wait_serial(['GNU C Library (GNU libc) stable release', 'Configured for i686-suse-linux.']) || die '/lib/libc.so.* did not match';
```
Comment (dzedro) [inline tests/console/glibc_i686.pm]: This is not the best way, but I was not successful to create regexp for this two lines and didn;t want to spend more time on it maybe somebody else can help

File: tests/console/glibc_i686.pm [hunk 2]
```diff
@@ -1,30 +1,28 @@
...
 #
 # Copying and distribution of this file, with or without modification,
 # are permitted in any medium without royalty provided the copyright
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# G-Summary: Rework the tests layout.
-# G-Maintainer: Alberto Planas <aplanas@suse.com>
+# Summary: Rework the tests layout.
+# Maintainer: Alberto Planas <aplanas@suse.com>
```
Comment (okurz) [inline tests/console/glibc_i686.pm]: please put proper content

File: tests/console/glibc_i686.pm (hunk 1 above)
Comment (okurz) [inline tests/console/glibc_i686.pm]: looks clean so I don't see a problem.

An alternative I could think of:

```perl
my $out = script_output '/lib/libc.so.*';
die '/lib/libc.so.* did not match' unless $out =~ /GNU C Library (GNU libc) stable release/;
die 'wrong libc version' $out =~ /Configured for i686-suse-linux./;
```

but that does not necessarily look nicer to me.

**Comment by dzedro** [comment]
Updated

**Comment by okurz** [comment]
LGTM

**Pull request merged**